unified-codes/libs/ui/src/lib/components/data/atoms/Table.stories.tsx

```diff
@@ -3,9 +3,9 @@ import { Table } from './Table';
 
 export default {
   component: Table,
-  title: 'Table',
+  title: 'Library/Table',
```

Comment:
Same as above! Since this is in the library scope, prepending the title with `Library` is quite generic and not sure what it adds (I know pot calling black on generic variable names!).

Reply:
Note that it also means that when the ui lib components are viewed with `nx storybook ui`, everything is under a generic `Library` root, which not sure if it adds much? Is there a way to map all the library titles in the web storybook config, maybe?